msi/GitProduct.wxs

```diff
@@ -6,26 +6,27 @@
   <?define SixtyFourBit ?>
 <?endif?>
 
-<Wix xmlns='http://wixtoolset.org/schemas/v4/wxs'>
-  <Product Id='*' Name='Git' Language='1033' Manufacturer='The Git Development Community'
+<Wix xmlns='http://wixtoolset.org/schemas/v4/wxs'
+     xmlns:swid='http://wixtoolset.org/schemas/v4/wxs/tag'>
+  <Product Id='*' Name='Git for Windows' Language='1033' Manufacturer='The Git Development Community'
            Version='$(var.Version)'
            UpgradeCode='abbb3e1d-fea0-4b84-9585-b4dc13cbc1b4'>
-    <Package InstallScope='perMachine' Compressed='yes' />
+    <Package InstallScope='perMachine' Compressed='yes' Keywords='Git' />
+    <swid:Tag Regid='git-for-windows.github.io' />
 
     <MajorUpgrade DowngradeErrorMessage='A newer version of [ProductName] is already installed.' />
     <MediaTemplate CabinetTemplate='git{0}.cab' CompressionLevel='high' EmbedCab='yes' />
 
+    <!-- Add/Remove Programs Properties -->
     <Property Id='ARPCONTACT' Value='https://github.com/git-for-windows/git/wiki/Contact' />
     <Property Id='ARPURLINFOABOUT' Value='https://git-for-windows.github.io/' />
     <Property Id='ARPNOMODIFY' Value='1' />
-
     <Property Id='ARPPRODUCTICON' Value='git.ico' />
-    <Icon Id='git.ico' SourceFile='git.ico' />
+    <SetProperty Id='ARPINSTALLLOCATION' Value='[INSTALLDIR]' After='CostFinalize' />
```

Comment:
Looks like I was wrong...during rebasing this got put back to `INSTALLDIR`. It'll be fixed in a bit.